package.json

```diff
@@ -75,6 +75,7 @@
     "history": "^4.7.2",
     "html-webpack-plugin": "^2.30.1",
     "husky": "^0.14.3",
+    "iocane": "^1.0.2",
```

Comment:
Do we need this anymore? because you are now using iocane from Buttercup right?

Reply:
Removed! 😃